backend/satellite_tools/download.py

```diff
@@ -138,11 +138,6 @@ def _do_perform(self):
                 self.parent.first_in_queue_done = True
                 self.parent.first_in_queue_lock.release()
 
-    def _set_opts(self, opts=None):
-        if not opts:
-            opts = {}
-        PyCurlFileObject._set_opts(self, opts=opts)
-        self.curl_obj.setopt(pycurl.FORBID_REUSE, 0) # pylint: disable=E1101
```

Comment:
Why is the reason of removing this block? Wouldn't we miss to do:

```python
self.curl_obj.setopt(pycurl.FORBID_REUSE, 0)
```

Reply:
I think we don't need that here if we pass `keepalive=True` as part of the options when we instantiate `PyCurlFileObjectThread`. 

```python
# urlgrabber/grabber.py
class PyCurlFileObject(object):
    def __init__(self, url, filename, opts):
        self.opts=opts
        # ...

    def _set_opts(self, opts={}):
        # XXX
        if not opts:
            opts = self.opts

        # keepalives
        if not opts.keepalive:
            self.curl_obj.setopt(pycurl.FORBID_REUSE, 1)
     
        # ... 
```
My understanding is that setting `FORBID_REUSE` to `0` is only needed because we let urlgrabber set it to `1` by not specifying `keepalive=True`. I've added that to the `opts` dict in line 215.

Curl's default is `0` ([docs](https://curl.se/libcurl/c/CURLOPT_FORBID_REUSE.html)), as long as we don't set it to `1` we should be fine.